### DISCUSSION TITLE
GroupedList - customize keyboard navigation

### DISCUSSION THREAD

**Discussion by qinzhouxu** [microsoft/fluentui#24078] [Q&A] [answered]
Hi folks,

For `GroupedList` control, the default behavior for keyboard navigation allows us to expand/collapse each level using left/right arrow keys. However, in order to meet consistent navigation requirement, our team need to enable user use Enter to do expand/collapse. Is there any method to achieve this?

**Comment by spmonahan** [comment, marked as answer]
This is not something currently supported in v8 though we are discussing for future v9 controls as this is fairly common request.

We don't have any plans to add full keyboard customization to GroupedList in v8 but if it's something you need I can help you work through it if you'd like to contribute the feature to Fluent UI.
